dbt/adapters/spark_cde/cdeapisession.py

```diff
@@ -22,16 +22,13 @@
 import time
 import traceback
 
-from dbt.adapters.spark_cde.adaptertimer import AdapterTimer
```

Comment:
Removing references to timer as it is polluting the output log. Instead we will have simple log statements before each method with job id and utc time so that trails can be easily follwed.

Reply:
Ok, this looks cleaner way to grep timing data.